portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,104 @@
+                        <br><b>Weaknesses:</b> Rubic's Cube Solving, Multi-tasking, FEA, chocolate mousse cakes    
+                    </p>
+                </section>
+            </div>
+
+            <!-- About my life 3 small articles-->
+            <section class="w3-row-padding w3-center w3-light-grey" id = "about-me">
+                <article class="w3-third">
+                
+                    <h3>My Path to Google</h3>
+                    <img src="./images/google2018.jpeg" alt="mypathtogoogle" style="width:100%">
+                    <p>In February 2019 my mum and I went to travel to USA. We did a road trip from Los Angeles to San Francisco and then to Las Vegas, but on the way to SF we visited San Jose to check out some science museums and Silicon Valley campuses. 
+                        Then we realised a tour can be arranged only by a company representative, who we did not know, so we just stared at it from the outside (lol). This is me on Google HQ Mountain View campus, I came there as a tourist and I told my mum I wanted to work there, she was skeptical. 
+                        It’s been almost 2 years since then and now I’m a STEP intern at Google Sydney branch. Dreams come true! 
+                    </p>
+                </article>
+
+                <article class="w3-third">
+                    <h3>My Hobbies</h3>
+                    <img src="./images/chess.jpeg" alt="Chess" style="width:100%">
+                    <p>I've been playing chess since primary school, and liked to compete. In my country chess is considered as a serious sport and I played my way up to the title of Candidate Master of Sports in Chess. Currently I don't practice, I only played a few South Australian championates and Australian Young Masters Open Tournament (got destroyed in both). My FIDE rating dropped to whooping 1588.</p>
+                </article>
+                
+                <article class="w3-third">
+                    <h3>My Vision</h3>
+                    <img src="./images/Screen Shot 2020-12-09 at 12.23.06 am.png" alt="me" style="width:100%">
+                    <p>I love about tech because it helps everyone and makes the world a better place every day. 
+                        I believe tech is beautiful, but it only is a tool, that makes things accessible and easy. It also is the most fast-moving and progressive industry, that is influencing everyone’s lives! So it has to be adaptive adn tolerant in terms of social norms, diversity and inclusion 🔭🚀
+                    </p>
+                </article>
+            </section>
+
+            <center>
+                <h4>Click here to get a random tv show suggestion from Jamila:</h4>
+                <button onclick="addRandomGreeting()">tv show?</button>
+                <div id="greeting-container"></div>
+            </center>
+            
+            
+            <!-- Table with my projects tbc -->
+
```

Comment:
Nitpick: No space between the block and the comment, that makes it clearer that the comment describes the block.

Reply:
fixed in the new pushed commit
also added a bit of content but no significant changes